services/mfa/mfa.go

```diff
@@ -112,7 +112,12 @@ func (m *Mfa) Deactivate(userId string) *model.AppError {
 	}
 
 	achan := m.Store.User().UpdateMfaActive(userId, false)
-	schan := m.Store.User().UpdateMfaSecret(userId, "")
+	schan := make(chan store.StoreResult, 1)
```

Comment:
Probably here we can just return a `*model.AppError` here instead of a `store.Result` actually it simplifies the usage later too.